.gitignore

```diff
@@ -235,7 +235,7 @@ assets/
 out
 .DS_Store
 .setup-done-before
-results/*
+results/*.mp4
```

Comment:
This is dysfunctional as the bot exports to `results/subreddit/file` rather than `results/file` and also the bot may export with a different filetype in the future and it is easier just to leave it as it is.